src/Symfony/Bridge/Doctrine/ManagerRegistry.php

```diff
@@ -55,8 +55,12 @@ function (&$wrappedInstance, LazyLoadingInterface $manager) use ($name) {
                 if (isset($this->aliases[$name])) {
                     $name = $this->aliases[$name];
                 }
-                $method = $this->methodMap[$name] ?? 'get'.strtr($name, $this->underscoreMap).'Service'; // BC with DI v3.4
-                $wrappedInstance = $this->{$method}(false);
+                if (isset($this->fileMap[$name])) {
+                    $wrappedInstance = $this->requireInScope($this->fileMap[$name], false);
+                } else { // BC with DI v3.4
+                    $method = $this->methodMap[$name] ?? 'get'.strtr($name, $this->underscoreMap).'Service';
```

Comment:
@nicolas-grekas you said on Slack that you can submit it on 3.4 but that version AFAIS has `php >=5.5.9` requirement, so `??` operator is not compatible. So it should be changed when submitting to 3.4 or maybe `@$this->methodMap[$name] ?:` should be used instead.